Bin/Documentation.php

```diff
@@ -309,7 +316,40 @@ public function main()
             }
         }
 
-        echo "\n", 'Open file://', $workspace, '/index.html', '.', "\n";
+        $pathname = escapeshellarg('file://' . $workspace . '/index.html');
+
+        echo "\n";
+
+        if (true === $open) {
+            if (isset($_SERVER['BROWSER'])) {
```

Comment:
Wow, thanks for the link 😄. They tell that $BROWSER can be a list of colon-delimited commands... Can this be a problem here ? Maybe we need some processing on this environment var before running it...